etc/config/c.amazon.properties

```diff
@@ -306,14 +316,19 @@ compiler.cmsp430g621.semver=6.2.1
 
 ################################
 # GCC for AVR
-group.cavr.compilers=cavrg454:cavrg464
+group.cavr.compilers=cavrg454:cavrg464:cavrg540
 group.cavr.groupName=AVR GCC
 group.cavr.baseName=AVR gcc
 group.cavr.isSemVer=true
 compiler.cavrg454.exe=/opt/compiler-explorer/avr/gcc-4.5.4/bin/avr-gcc
 compiler.cavrg454.semver=4.5.4
 compiler.cavrg464.exe=/opt/compiler-explorer/avr/gcc-4.6.4/bin/avr-gcc
 compiler.cavrg464.semver=4.6.4
+compiler.avrg540.exe=/opt/compiler-explorer/avr/arduino-1.8.9/hardware/tools/avr/bin/avr-g++
```

Comment:
This needs to be `compiler.cavrg540` please. Again, apologies for the tricky configuration syntax :(